pyartifactory/models/Auth.py

```diff
@@ -16,3 +16,11 @@ class ApiKeyModel(BaseModel):
 
 class PasswordModel(BaseModel):
     password: SecretStr
+
+
+class AccessTokenModel(BaseModel):
+    access_token: str
+    expires_in: Optional[int] = 3600
+    scope: Optional[str] = None
+    refresh_token: Optional[bool] = False
+    token_type: str

```

Comment:
If I read the [documentation](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-CreateToken), Artifactory always provides the `api:*` scope, so I guess `scope` wouldn't be Optional.
Also the `refresh_token` is a string if it exists, so it should be defined as `Optional[str] = None`.

Reply:
indeed scope will get populated with api:* always at least. 
fixed the other as well.

Reply:
as for the objects.py.... I don't know, I run a mac and edit exclusively in pycharm, so I don't really understand how I could have got into line ending weirdness at all honestly.